docs/code_formatting_rules.md

```diff
@@ -180,8 +180,7 @@ like `require.NoErrorf()`.
 
 ### Wrapping long function definitions
 
-If one is forced to wrap lines of function arguments that exceed the 80
-character limit, then indentation must be kept on the following lines. Also,
+If one is forced to wrap lines of function arguments that exceed the 80-character limit, then indentation must be kept on the following lines. Also,
```

Comment:
hmmm unfortunately this change breaks the rule being noted 🤔  we want to keep the 80-char wrapping for now